docs/resources/team.md

```diff
@@ -23,29 +23,32 @@ user roles.
 
 ### Required
 
-- `name` (String) The name of this team.
+- `name` (String) The name of this resource.
 
 ### Optional
 
-- `can_be_deleted` (Boolean)
-- `can_be_renamed` (Boolean)
-- `can_change_members` (Boolean)
-- `can_change_roles` (Boolean)
-- `description` (String) The user-friendly description of this team.
+- `can_be_deleted` (Boolean, Deprecated)
+- `can_be_renamed` (Boolean, Deprecated)
+- `can_change_members` (Boolean, Deprecated)
+- `can_change_roles` (Boolean, Deprecated)
+- `description` (String) The description of this team.
```

Comment:
This change arises from helper methods used by the new resources... hopefully users won't think we've decided descriptions don't need to be user-friendly anymore 😄 . I can manually change it so that it's consistent with the old value, if anyone feels strongly about it?